### ISSUE TITLE
Problem with v6

### ISSUE THREAD

**Issue by throrin19** [mysticatea/vue-eslint-parser#41]
I have many problems with the version 6.x of this package and the eslint-plugin-vue part.

With the v5, I have no eslint error in my code.
With the v6, I have many eslint errors and warnings in my code : 
   - the error `vue/require-component-is` but my components have the `:is` 

```vue
<component
    :is="header.type"
    :header="header"
    :item="props.item"
    class="text-xs-right"
    @click="launchAction(header.event, props.item)" />
```

  - the `vue/valid-template-root` in v-if/v-else case
  - the `vue/no-unused-components` in case of use them in `<component>`
  - order of attributes
  - other warnings / errors that would not normally trigger

I don't understand why all works correctly in v5 bu not in v6 of your package

**Comment by mysticatea** [comment]
Because major version contains breaking changes. `eslint-plugin-vue` does not support v6 yet.
